ferrocene/doc/safety-manual/src/rustfmt/constraints.rst

```diff
@@ -0,0 +1,15 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+Constraints
```

Comment:
Constraints should be styled like rustc constraints: a section for each constraint, with a unique ID for each constraint and linking to the nodes of the HazOp they address.